### ISSUE TITLE
error import shogun using root account

### ISSUE THREAD

**Issue by Erichliu00** [shogun-toolbox/shogun#3763]
Hi, are there any instructions on how to correctly set up shogun on ubuntu 14.04 using root account. 
For some reason, I was using root account and tried to install shogun follows [INSTALL.md](https://github.com/shogun-toolbox/docs/blob/master/INSTALL.md#ubuntu) but later when using Shogun I met import errors: `ImportError: No module named modshogun `. Thank you.

**Comment by karlnapf** [comment]
Yep, exactly in that file.
With this description we won't be able to help you.

**Comment by Erichliu00** [comment]
Thanks, I retried this description but always got:

> Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package python-shogun

What should I do? I searched google and found [this similar question](http://stackoverflow.com/questions/42928900/could-not-install-sudo-apt-get-install-libshogun17) but it not works for me. I am using ubuntu 14.04. Thank you for any suggestions.
